lua/acf/core/utilities/util_sv.lua

```diff
@@ -303,12 +355,17 @@ do -- Entity linking
 		return Result
 	end
 
+	-- Returns the link source callable of a given class and VarName
+	---@param Class string The name of the class
+	---@param VarName string The varname for the given class
+	---@return fun(Entity:table):table # The link source callable
 	function ACF.GetLinkSource(Class, VarName)
 		if not EntityLink[Class] then return end
 
 		return EntityLink[Class][VarName]
 	end
 
+	-- Returns a table of entities linked to the given entity.
```

Comment:
Is this intended to be documented without annotations?

Reply:
No, that was a mistake. Fixed.